src/PureCloud.Client/Apis/WebDeploymentsApi.cs

```diff
@@ -0,0 +1,363 @@
+using System.Collections.Specialized;
+using System.Net.Http.Json;
+using Microsoft.Extensions.Options;
+using PureCloud.Client.Contracts;
+using PureCloud.Client.Http;
+using PureCloud.Client.Json;
+using PureCloud.Client.Models;
+
+namespace PureCloud.Client.Apis;
+
+/// <inheritdoc />
+public sealed class WebDeploymentsApi : IWebDeploymentsApi
+{
+    private readonly HttpClient _httpClient;
+    private readonly PureCloudJsonSerializerOptions _options;
+
+    public WebDeploymentsApi(IHttpClientFactory httpClientFactory, IOptions<PureCloudJsonSerializerOptions> options)
+    {
+        _httpClient = httpClientFactory.CreateClient(PureCloudConstants.PureCloudClientName);
+        _options = options.Value;
+    }
+
+    /// <inheritdoc />
+    public async Task<bool> DeleteWebdeploymentsConfigurationAsync(string configurationId, CancellationToken cancellationToken = default)
+    {
+        ArgumentException.ThrowIfNullOrEmpty(configurationId);
+
+        var uri = $"/api/v2/webdeployments/configurations/{Uri.EscapeDataString(configurationId)}";
```

Comment:
You can remove the variable by moving the string into the DeleteAsync method directly